src/main/java/io/temporal/samples/hello/HelloActivity.java

```diff
@@ -74,39 +104,80 @@ public String getGreeting(String name) {
     }
   }
 
+  /**
+   * Implementation of our workflow activity interface. It overwrites our defined composeGreeting
```

Comment:
I would rephrase:
"Simple activity implementation, that concatenates two strings."